test/client/fixtures/runner/runner-test.js

```diff
@@ -1,35 +1,63 @@
-var hammerhead  = window.getTestCafeModule('hammerhead');
+var hammerhead = window.getTestCafeModule('hammerhead');
 var hhsettings = hammerhead.get('./settings').get();
 
 var testCafeCore = window.getTestCafeModule('testCafeCore');
 var COMMAND      = testCafeCore.COMMAND;
 var SETTINGS     = testCafeCore.get('./settings').get();
 var transport    = testCafeCore.get('./transport');
+var ERROR_TYPE   = testCafeCore.ERROR_TYPE;
 
 var testCafeRunner = window.getTestCafeModule('testCafeRunner');
 var actionBarrier  = testCafeRunner.get('./action-barrier/action-barrier');
 var Runner         = testCafeRunner.get('./runner');
 
 
-transport.batchUpdate                = function (callback) {
+var runner                        = null;
+var savedTakeScreenshotOnFails    = null;
+var savedAsyncServiceMsg          = null;
+var savedTransportFatalError      = null;
+var savedTransportAssertionFailed = null;
+
+transport.batchUpdate = function (callback) {
     callback();
 };
+
 actionBarrier.waitPageInitialization = function (callback) {
     callback();
 };
-$.fn.load                            = function (callback) {
+
+$.fn.load = function (callback) {
     callback();
 };
 
 Runner.checkStatus = function () {
 };
 
+QUnit.testStart(function () {
+    testRunner                    = new Runner();
+    savedTakeScreenshotOnFails    = SETTINGS.TAKE_SCREENSHOT_ON_FAILS;
```

Comment:
The `SETTINGS.TAKE_SCREENSHOT_ON_FAILS` setting is not changed in the tests. Saving it is redundant